food/ios/App/Detail/DetailViewController.swift

```diff
@@ -264,6 +271,13 @@ class DetailViewController: UIViewController, Contextual {
     }
   }
 
+  @IBAction func openInMap() {
+    let app = AppDelegate.shared()
+    if let link = self.place.appleMapURL {
```

Comment:
Those with google maps installed will prefer it; let's detect and use if available.